sdk/python/assets/data/data.ipynb

```diff
@@ -221,6 +221,33 @@
     "returned_job.studio_url"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Authenticate with user identity\r\n",
+    "\r\n",
+    "When running a job on a compute cluster, you can also use your user identity to access data. To enable job to access data on behald of you, specify **identity= AzureMLOnBehalfOfCredential()** in job definition, as shown below. For more details, see [Accessing storage services](https://learn.microsoft.com/azure/machine-learning/how-to-identity-based-service-authentication)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ml.Identity import AzureMLOnBehalfOfCredential\r\n",
+    "\r\n",
+    "job = command(\r\n",
+    "    code=\"./src\",\r\n",
+    "    command=\"python read_data.py --input_data ${{inputs.input_data}}\",\r\n",
+    "    inputs=my_job_inputs,\r\n",
+    "    environment=\"AzureML-sklearn-1.0-ubuntu20.04-py38-cpu:1\",\r\n",
+    "    compute=\"cpu-cluster\",\r\n",
+    "    identity= AzureMLOnBehalfOfCredential() \r\n",
```

Comment:
Think we should use UserIdentity object here:  https://learn.microsoft.com/en-us/python/api/azure-ai-ml/azure.ai.ml.useridentity?view=azure-python-preview

Reply:
Fixed